src/tests/components/ForecastBreakdown.test.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastBreakdown from "../../components/ForecastBreakdown";
+
+describe("ForecastBreakdown", () => {
+  const validProps = {
+    icon: "200",
+    temp: "20",
+    minTemp: "18",
+    maxTemp: "22",
+    humidity: "30",
+    windSpeed: "40",
+    weather: "cold",
+  };
+  const { icon, temp, minTemp, maxTemp, humidity, windSpeed, weather } =
```

Comment:
This seems a bit redundant, creating a valid props object only to destructure it straight away. I would just reference the valid props properties in the tests below directly eg `icon={validProps.icon}`